### DISCUSSION TITLE
Module not found Errors [Solved]

### DISCUSSION THREAD

**Discussion by 87prashant** [ParabolInc/parabol#6774] [Q&A] [answered]
I am started getting "module not found" errors, don't know what causes this. I never changed these files.
Any help

![Screenshot 2022-06-16 183333](https://user-images.githubusercontent.com/106697681/174087232-2d9f3c01-d1cb-4209-a76b-fa081a0543ce.png)

**Comment by Dschoordsch** [comment]
Do these go away? Are these still happening the second time you start the app?

When starting the app we run multiple services in parallel to speed up the startup time. Some of these are generating files (Relay compiler creates `.graphql` from inline fragments for example). When the web service is faster than the compiler, it will show this message, but it watches the directory where the files are generated, so once they are, it should restart.
It's not the best developer experience, bot neither is slow startup time. We're open to suggestions!

**Reply by 87prashant** [reply]
No these are not happening anymore. Maybe they were happening because I messed up the local repo with commits. I fixed the local repo and errors also went away.




But I am having the below errors, is this normal? 

I just followed the installation guide on readme.md
`$ git clone https://github.com/ParabolInc/parabol.git`
`$ cd parabol`
`$ cp .env.example .env # Add your own vars here`
`$ yarn`
`$ yarn db:start`
`$ yarn dev `

or am I missing something?

![Screenshot 2022-06-17 005641](https://user-images.githubusercontent.com/106697681/174150368-3be2ee2a-cdf0-40d9-8743-887b129de34e.png)

**Comment by 87prashant** [comment, marked as answer]
>No these are not happening anymore. Maybe they were happening because I messed up the local repo with commits. I fixed the local repo and errors also went away.

This worked for me
